Gemfile

```diff
@@ -5,19 +5,19 @@ source 'https://rails-assets.org'
 gem 'tzinfo-data', platforms: [:mswin]
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 4.1.8'
+gem 'rails', '~> 4.2.0'
 
 # Use PostgreSQL for the backend
 gem 'pg'
 # Schema Plus for some higher level database abstractions
-gem 'schema_plus', github: 'lowjoel/schema_plus', branch: '1.x'
+gem 'schema_plus', github: 'lowjoel/schema_plus'
 gem 'schema_validations'
 # Table and column comments
 gem 'migration_comments'
 # Add creator_id and updater_id attributes to models
 gem 'magiclabs-userstamp', github: 'lowjoel/userstamp'
 # Squeel as an SQL-like DSL
-gem 'squeel'
+gem 'squeel', github: 'danielrhodes/squeel'
```

Comment:
Few guys have fixed the issue in their own repo, the one I used is from one of them.